Build fails: error: 'runtime_error' is not a member of 'std'

## Describe the Bug
I cannot build. The build stops with
```
[ 13%] Building CXX object src/cs-graphics/CMakeFiles/cs-graphics.dir/internal/tiny_gltf_helper.cpp.o
/home/mcd/code/ap/cosmoscout-vr/src/cs-graphics/internal/tiny_gltf_helper.cpp: In function 'tinygltf::Image cs::graphics::internal::loadImage(const string&)':
/home/mcd/code/ap/cosmoscout-vr/src/cs-graphics/internal/tiny_gltf_helper.cpp:72:16: error: 'runtime_error' is not a member of 'std'
   72 |     throw std::runtime_error(std::string("loadImage: Unable to load ") + filepath);
      |                ^~~~~~~~~~~~~
make[2]: *** [src/cs-graphics/CMakeFiles/cs-graphics.dir/build.make:174: src/cs-graphics/CMakeFiles/cs-graphics.dir/internal/tiny_gltf_helper.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:603: src/cs-graphics/CMakeFiles/cs-graphics.dir/all] Error 2
make: *** [Makefile:150: all] Error 2
```
There are other files with the same problem.

Steps to reproduce the behavior:
1. Build on linux with gcc-10.1 by using make_externals.sh and make.sh as noted in install.md
2. See error

The gcc invocation is
`/usr/bin/c++ -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DBOOST_CHRONO_DYN_LINK -DBOOST_DATE_TIME_DYN_LINK -DBOOST_FILESYSTEM_DYN_LINK -DDOCTEST_CONFIG_DISABLE -DGLM_ENABLE_EXPERIMENTAL -DGLM_FORCE_SWIZZLE -DNOMINMAX -Dcs_graphics_EXPORTS -I/home/mcd/code/ap/cosmoscout-vr/build/linux-release/src/cs-core -I/home/mcd/code/ap/cosmoscout-vr/build/linux-release/src/cs-utils -I/home/mcd/code/ap/cosmoscout-vr/build/linux-release/src/cs-graphics -I/home/mcd/code/ap/cosmoscout-vr/build/linux-release/src/cs-gui -I/home/mcd/code/ap/cosmoscout-vr/build/linux-release/src/cs-scene -I/home/mcd/code/ap/cosmoscout-vr/install/linux-externals-release/lib/DriverPlugins -isystem /home/mcd/code/ap/cosmoscout-vr/externals/gli -isystem /home/mcd/code/ap/cosmoscout-vr/install/linux-externals-release/include -isystem /home/mcd/code/ap/cosmoscout-vr/externals/doctest -isystem /home/mcd/code/ap/cosmoscout-vr/externals/tinygltf -isystem /home/mcd/code/ap/cosmoscout-vr/install/linux-externals-release/include/OpenSG -O3 -DNDEBUG -fPIC -std=gnu++17 -o CMakeFiles/cs-graphics.dir/internal/tiny_gltf_helper.cpp.o -c /home/mcd/code/ap/cosmoscout-vr/src/cs-graphics/internal/tiny_gltf_helper.cpp`

The problem is reproducible with this testprogram 

```
int main() {
        throw std::runtime_error("some runtime error");
}
```
Adding `#include <stdexcept>` fixes compilation.

## System
_Please complete the following information:_
 - CosmoScout VR version v1.2.0
 - OS: artix linux (archlinux derivate)
 - Compiler: gcc-10.1.0

--- a/src/cs-graphics/HDRBuffer.hpp
+++ b/src/cs-graphics/HDRBuffer.hpp
@@ -10,6 +10,7 @@
 #include "cs_graphics_export.hpp"
 
 #include <array>
+#include <cstdint>
 #include <unordered_map>
 #include <vector>
 

--- a/src/cs-gui/types.hpp
+++ b/src/cs-gui/types.hpp
@@ -11,6 +11,7 @@
 
 #include <cstdint>
 #include <functional>
+#include <string>
 #include <variant>
 #include <vector>
 

